source/less/blocks/about.less

```
@@ -1,29 +1,160 @@
 .about {
+  position: relative;
   box-sizing: border-box;
-  padding: 32px 60px 66px 50px;
+  padding: 30px 60px 66px 50px;
   color: @basic-white;
   background: @main-background url("../../img/about-mobile.png") no-repeat;
   background-position: left 20px bottom;
   .container-round-bottom();
+
+  @media (min-width: @tablet-width) {
+    padding: 36px 391px 0 75px;
+    background: @header-color url("../../img/about-table-bg.png") no-repeat;
```

Здесь и в остальных случаях с фоновыми картинками: есть лишний переход наверх - `../`. Этот набор символов в относительном пути к файлу означает, что нужно подняться на один уровень вверх относительно текущего пути. Файл стилей находится в каталоге **css** и один уровень вверх относительно этого каталога - корень сайта, но если попытаться дважды подняться вверх, на что указывает `../../`, то там ничего нет, так как за пределы корневого каталога мы подняться не можем. На локальном сервере всё выглядит хорошо и эта ситуация обрабатывается, но на реальном сервере может появиться проблема - опубликуйте задание и увидите, что в Github Pages картинка не будет отображаться, так как браузер её не найдёт.

Должно быть так:
```suggestion
    background: @header-color url("../img/about-table-bg.png") no-repeat;
```